.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+48819da8bb9d1b7fbc2911c3865f2188ee0400bb
```

Comment:
@bengioe once we squash and merge this, we will update this value to the sha of that merge commit.